extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -455,7 +467,11 @@ Type attribute value: `piStats`
 
 Type attribute value: `pressboard`
 
-Adds support for Pressboard. Configuration details can be found at [help.pressboard.ca] `http://help.pressboard.ca/publisher-resources/getting-started/implementing-google-amp`.
+<!--
+(Link 404s, unclear whether vendor is still supported.)
```

Comment:
Hmm, maybe we should see who added `pressboard` & try to ping them and see if they are in fact deprecated.

Reply:
We can try, but we should do that after this change. This doesn't remove them from the list, just removes the link blurb.